src/metainfo/yast2-tune.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Dominique Leuenberger <dimstar@opensuse.or> -->
+<component type="addon">
+<id>yast2-tune</id>
+<extends>yast2.desktop</extends>
+<_name>Hardware Information</_name>
```

Comment:
Ups - I prepared the file to pass through the translator, but that is not integrated into the build system... the easiest would be to pull in appstream.m4, which has all the logic, but it might be a bit excessive